core/client/fs/src/test/java/alluxio/client/file/BaseFileSystemTest.java

```diff
@@ -447,4 +450,39 @@ public void unmountException() throws Exception {
       assertSame(EXCEPTION, e);
     }
   }
+
+  /**
+   * Ensures Warnings are logged or exceptions are thrown when an {@link AlluxioURI} with a
```

Comment:
does this test warnings being logged?

Reply:
I need to work out a way to do that

Reply:
Could try something like this? https://stackoverflow.com/a/3717461/401884

I don't know if we ever test log warnings, but I think it would be quite useful, especially for a feature like this